test/unit/identifiers.spec.ts

```diff
@@ -845,3 +845,22 @@ test("lua built-in as in constructor assignment", () => {
         export const result = new A("42").error;
     `.expectToMatchJsResult();
 });
+
+test("customName", () => {
+    const result = util.testModule`
+        /** @customName test2 **/
+        function test(this: void) {}
+        test()
+    `.getLuaResult();
```

Comment:
The tests should execute the generated code and expect some kind of result, to show that it works.

Reply:
```ts
const result = util.testModule`
        /** @customName test2 **/
        function test(this: void) { return 3; }
        export const result = test();
    `
    .expectToEqual({ result: 3 })
    .getLuaResult();
 ```

Reply:
Ah, but that test would still be valid if the function name didn't even change though?

Reply:
Yeah also keep the rest, just add the expect for the execution result so we test it executes correctly

Reply:
> Yeah also keep the rest, just add the expect for the execution result so we test it executes correctly

Ahhh, okay, great, thank you.
I'll do that and add some more tests then. (based on what you noted above)